frame/system/src/mock.rs

```diff
@@ -108,26 +97,27 @@ impl Config for Test {
 	type AccountId = u64;
 	type Lookup = IdentityLookup<Self::AccountId>;
 	type Header = Header;
-	type Event = Event<Self>;
+	type Event = Event;
 	type BlockHashCount = BlockHashCount;
 	type DbWeight = DbWeight;
 	type Version = Version;
-	type PalletInfo = ();
+	type PalletInfo = PalletInfo;
 	type AccountData = u32;
 	type OnNewAccount = ();
 	type OnKilledAccount = RecordKilled;
 	type SystemWeightInfo = ();
 	type SS58Prefix = ();
 }
 
-pub type System = Module<Test>;
-pub type SysEvent = <Test as Config>::Event;
+pub type SysEvent = frame_system::Event<Test>;
 
-pub const CALL: &<Test as Config>::Call = &Call;
+/// A simple call, which one doesn't matter.
+pub const CALL: &<Test as Config>::Call = &Call::System(frame_system::Call::set_heap_pages(0u64));
```

Comment:
previously Call was a strut which panics in its implementation of dispatchable and was meant to be unused.
Using a random call doesn't change the nature of the test